### PR TITLE
Add proxysql blog #one

### PR DESCRIPTION
**Pull request by TasdidurRahman**
Signed-off-by: TasdidurRahman <tasdid@appscode.com>

### CODE REVIEW

File: content/post/proxysql-one-2022.06.01/index.md [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+| proxysql      |
+| repl          |
+| root          |
+| mysql.session |
+| mysql.sys     |
+| root          |
++---------------+
+7 rows in set (0.00 sec)
+```
+note : the use `proxysql` is the monitor user for ProxySQL created by KubeDB operator
```
Comment (DRoyyy) [inline content/post/proxysql-one-2022.06.01/index.md:247]: note isn't clear
and use '>' that symbol before any kind of note, Like >Note:

File: content/post/proxysql-one-2022.06.01/index.md [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+
+mysql> flush privileges;
+Query OK, 0 rows affected (0.01 sec)
+```
+
+#### <a id="user-entry"></a> Put entry for test user in ProxySQL
+
+* Exec into any ProxySQL pod with the following command, we will end up with something like this,
+```shell
+~ $ ~ $ kubectl exec -it -n demo proxy-mysql-server-0 -- bash
```
Comment (DRoyyy) [inline content/post/proxysql-one-2022.06.01/index.md:287]: remove ~$ ~

File: content/post/proxysql-one-2022.06.01/index.md [hunk 3]
```diff
@@ -0,0 +1,536 @@
...
+
+#### Install KubeDB
+
+We need to have KubeDB operator version `v2022.05.24` or later to test ProxySQL. To install KubeDB in your cluster, you may follow this [Link](https://kubedb.com/docs/v2022.05.24/setup/)
+
+#### Install cert manager
+
+If you don't want TLS secured connection for your ProxySQL or MySQL you can skip this part. As we are going to test TLS secured connections, we need to install cert manager in our cluster first. You can install cert manager operator in your cluster with this following command, <br>
+```shell
+~$ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.2/cert-manager.yaml
```
Comment (DRoyyy) [inline content/post/proxysql-one-2022.06.01/index.md:47]: don't have to use ~ that symbol

File: content/post/proxysql-one-2022.06.01/index.md [hunk 4]
```diff
@@ -0,0 +1,536 @@
...
+mysql> grant all privileges on random.* to 'test'@'%';
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> flush privileges;
+Query OK, 0 rows affected (0.01 sec)
+```
+
+#### <a id="user-entry"></a> Put entry for test user in ProxySQL
+
+* Exec into any ProxySQL pod with the following command, we will end up with something like this,
```
Comment (DRoyyy) [inline content/post/proxysql-one-2022.06.01/index.md:285]: Exec into any ProxySQL pod and Login to the ProxySQL admin console

File: content/post/proxysql-one-2022.06.01/index.md [hunk 5]
```diff
@@ -0,0 +1,536 @@
...
+```shell
+~ $ ~ $ kubectl exec -it -n demo proxy-mysql-server-0 -- bash                  
+root@proxy-mysql-server-0:/#
+```
+* Log into the proxysql admin console.
```
Comment (DRoyyy) [inline content/post/proxysql-one-2022.06.01/index.md:290]: remove that section header

**Review by DRoyyy — CHANGES_REQUESTED** [review]

File: content/post/proxysql-one-2022.06.01/index.md (hunk 5 above)
Comment (DRoyyy) [reply content/post/proxysql-one-2022.06.01/index.md:290]: yes

File: content/post/proxysql-one-2022.06.01/index.md (hunk 1 above)
Comment (TasdidurRahman) [reply content/post/proxysql-one-2022.06.01/index.md:247]: fixed

File: content/post/proxysql-one-2022.06.01/index.md (hunk 3 above)
Comment (TasdidurRahman) [reply content/post/proxysql-one-2022.06.01/index.md:47]: removed

File: content/post/proxysql-one-2022.06.01/index.md (hunk 4 above)
Comment (TasdidurRahman) [reply content/post/proxysql-one-2022.06.01/index.md:285]: merged

File: content/post/proxysql-one-2022.06.01/index.md (hunk 5 above)
Comment (TasdidurRahman) [reply content/post/proxysql-one-2022.06.01/index.md:290]: merged

**Review by DRoyyy — APPROVED** [review]

**Review by spectro30 — DISMISSED** [review]
LGTM

**Pull request merged**